apps/platform/src/app/(main)/(project)/[workspace]/[project]/@environment/page.tsx

```diff
@@ -39,24 +39,32 @@ function EnvironmentPage(): React.JSX.Element {
   )
 
   useEffect(() => {
-    if (!selectedProject) return 
-    setLoading(true)
+  if(!selectedProject) return
```

Comment:
You can remove this check since its already present in `handleEnvironmentsFetch`